appinventor/blocklyeditor/src/blocks/components.js

```diff
@@ -340,6 +347,13 @@ Blockly.Blocks.component_method = {
       //var title = this.inputList[0].titleRow[0];
       //title.setText('call ' + this.instanceName + '.' + this.methodType.name);
       this.componentDropDown.setValue(this.instanceName);
+      if (this.isCollapsed())
+      {
```

Comment:
Don't use C# curly brackets style here (and all if statements). Place them in the same line, as the rest of the code.